source/Calamari.Common/Features/Behaviours/ConfigurationVariablesBehaviour.cs

```diff
@@ -17,13 +18,20 @@ public class ConfigurationVariablesBehaviour : IBehaviour
         readonly IVariables variables;
         readonly IConfigurationVariablesReplacer replacer;
         readonly ILog log;
+        private readonly string subdirectory;
 
-        public ConfigurationVariablesBehaviour(ICalamariFileSystem fileSystem, IVariables variables, IConfigurationVariablesReplacer replacer, ILog log)
+        public ConfigurationVariablesBehaviour(
+            ICalamariFileSystem fileSystem,
+            IVariables variables,
+            IConfigurationVariablesReplacer replacer,
+            ILog log,
+            string? subdirectory = "")
```

Comment:
I'm noticing the `?` after `string`. Since this arg is default to `""`, can it really be `null`? If it is `null`, then `Path.Combine` will throw an Exception I think.

Reply:
Ah good catch, yep I think originally I had it as `null` default but changed it precisely because Path.Combine will take an empty string.